docs/admin/Permissions/permissions.md

```diff
@@ -0,0 +1,23 @@
+# Permissions
+
+Hubspot Super Admins have access to create a new user and change user permission settings throughout. There are no permission roles in the Cal-ITP Hubspot account, and permissions determined by the Super Admins.
```

Comment:
This line is confusing to me but not sure what the edit would be--
`and permissions determined by the Super Admins.`

Reply:
Updated the language on this.